tools/register_users_from_csv.py

```diff
@@ -58,7 +58,7 @@ def create_user(self, username, first_name, last_name, email):
         r = self.session.get(url)
         if r.status_code == 200:
             print(f"User {username} exists.")
-        if r.status_code == 404:
+        if r.status_code in [404, 500]:
```

Comment:
This is in reference to #21 that has been fixed but the fix hasn't rolled out yet.